sisyphus/global_settings.py

```diff
@@ -172,6 +172,8 @@ def file_caching(path):
 
 #: set true to automatically clean jobs in error state and retry
 CLEAR_ERROR = False
+
```

Comment:
Add a comment starting with '#:' before PRINT_ERROR, otherwise it will not show up in the sphinx documentation.